elements/pfe-sass/variables/_maps.scss

```diff
@@ -53,6 +53,8 @@ $pfe-colors: (
   ui-disabled--text:                  $pfe-color--ui-disabled--text,
   ui-disabled--text--hover:           $pfe-color--ui-disabled--text--hover,
 
+  ui-tab--dark-hover-color:           $pfe-color--white,
```

Comment:
Does this need to be moved into the component code too?  I don't see other ui-tab variables here that this goes with.